CONTRIBUTING.md

```diff
@@ -112,3 +108,45 @@ If that works, proceed with releasing these crates.
 [chat]: https://docs.filecoin.io/about-filecoin/chat-and-discussion-forums/#chat
 [discuss]: https://github.com/filecoin-project/fvm-specs/discussions
 [issue]: https://github.com/filecoin-project/ref-fvm/issues
+
+### Release Process
+
+This section describes the automated parts of the release process and the manual steps FVM owners need to take.
+
+#### Current State
+
+1. On a release pull request creation, a [Release Checker](.github/workflows/release-check.yml) workflow will run. It will perform the following actions:
```

Comment:
```suggestion
1. Create a pull request which updates the [`version` in one or more `Cargo.toml` files](https://github.com/search?q=repo%3Afilecoin-project%2Fref-fvm+path%3ACargo.toml+%2F%5Eversion+%3D%2F&type=code).
   - Title the PR `chore: release vX.Y.Z`
2. On such a release PR's creation, a [Release Checker](.github/workflows/release-check.yml) workflow will run. It will perform the following actions:
```